<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DishInfoBar</class>
 <widget class="QWidget" name="DishInfoBar">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>447</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>菜品图片：</string>
   </property>
   <property name="buddy">
    <cstring>dishImagePathSelect_ptn</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>菜品名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>菜品价格：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>菜品剩余数量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="setDishImagePathSelect_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
   background-color: #ffffff; /* 设置背景颜色为白色 */
   border: 1px solid #b3b3b3; /* 设置边框为灰色 */
   padding: 8px; /* 设置内边距 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}

/* 悬停样式 */
QLineEdit:hover {
   border: 1px solid #808080; /* 设置悬停时的边框颜色为深灰色 */
}

/* 选中样式 */
QLineEdit:focus {
   border: 1px solid #0000ff; /* 设置选中时的边框颜色为蓝色 */
}

</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="dishImagePathSelect_ptn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color: #FFD1DC; /* 设置背景颜色为淡粉色 */
   color: #ffffff; /* 设置文字颜色为白色 */
   border: none; /* 去除边框 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}
   /* 悬停样式 */
   QPushButton:hover {
       background-color: #FFB6C1; /* 设置悬停时的背景颜色 */
   }

   /* 选中样式 */
   QPushButton:pressed {
       background-color: #FF69B4; /* 设置选中时的背景颜色 */
   }</string>
   </property>
   <property name="text">
    <string>选取菜品图片</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="setDishName_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
   background-color: #ffffff; /* 设置背景颜色为白色 */
   border: 1px solid #b3b3b3; /* 设置边框为灰色 */
   padding: 8px; /* 设置内边距 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}

/* 悬停样式 */
QLineEdit:hover {
   border: 1px solid #808080; /* 设置悬停时的边框颜色为深灰色 */
}

/* 选中样式 */
QLineEdit:focus {
   border: 1px solid #0000ff; /* 设置选中时的边框颜色为蓝色 */
}

</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="setDishMoney_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
   background-color: #ffffff; /* 设置背景颜色为白色 */
   border: 1px solid #b3b3b3; /* 设置边框为灰色 */
   padding: 8px; /* 设置内边距 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}

/* 悬停样式 */
QLineEdit:hover {
   border: 1px solid #808080; /* 设置悬停时的边框颜色为深灰色 */
}

/* 选中样式 */
QLineEdit:focus {
   border: 1px solid #0000ff; /* 设置选中时的边框颜色为蓝色 */
}

</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="setDishNum_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
   background-color: #ffffff; /* 设置背景颜色为白色 */
   border: 1px solid #b3b3b3; /* 设置边框为灰色 */
   padding: 8px; /* 设置内边距 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}

/* 悬停样式 */
QLineEdit:hover {
   border: 1px solid #808080; /* 设置悬停时的边框颜色为深灰色 */
}

/* 选中样式 */
QLineEdit:focus {
   border: 1px solid #0000ff; /* 设置选中时的边框颜色为蓝色 */
}

</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="commit_ptn">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>400</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color: #4CAF50; /* 设置背景颜色为绿色 */
   color: #ffffff; /* 设置文字颜色为白色 */
   border: none; /* 去除边框 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}
   /* 悬停样式 */
   QPushButton:hover {
       background-color: #45a049; /* 设置悬停时的背景颜色 */
   }

   /* 选中样式 */
   QPushButton:pressed {
       background-color: #3e8e41; /* 设置选中时的背景颜色 */
   }
</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_ptn">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>400</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
   background-color: #ffcc00; /* 设置背景颜色为黄色 */
   color: #ffffff; /* 设置文字颜色为白色 */
   border: none; /* 去除边框 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}
   /* 悬停样式 */
   QPushButton:hover {
       background-color: #e6b800; /* 设置悬停时的背景颜色 */
   }

   /* 选中样式 */
   QPushButton:pressed {
       background-color: #cca300; /* 设置选中时的背景颜色 */
   }</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteDish_ptn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color: #ff4d4d; /* 设置背景颜色为红色 */
   color: #ffffff; /* 设置文字颜色为白色 */
   border: none; /* 去除边框 */
   border-radius: 5px; /* 设置边框为5像素的圆角 */
}
   /* 悬停样式 */
   QPushButton:hover {
       background-color: #e64040; /* 设置悬停时的背景颜色 */
   }

   /* 选中样式 */
   QPushButton:pressed {
       background-color: #cc3333; /* 设置选中时的背景颜色 */
   }
</string>
   </property>
   <property name="text">
    <string>删除菜品</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>34</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>菜品选项卡</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
